database.example.js

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think this should be `database.example.json`: 

> Location of the database.json file.             [default: "./database.json"]

from https://db-migrate.readthedocs.io/en/latest/Getting%20Started/installation/

Reply:
:+1: Ah, yes, thank you.